docs/dev-guide/configuration.md

```diff
@@ -0,0 +1,773 @@
+---
+id: dev-guide-configuration
+title: Configuration
+---
+
+:::note NOTE
+Options marked with ***** are not overwritable through `configOverwrite`
```

Comment:
Do we have any of these yet? Can we use something mroe visual perhaps? Like 🚫 or ❌

Reply:
We do. Will update to use 🚫